src/test/scala/chiselTests/BundleElementsSpec.scala

```diff
@@ -44,27 +44,27 @@ class BpipDecoupled extends BpipOneField {
 class HasDecoupledBundleByInheritance extends Module {
```

Comment:
add more tests that mess with object creation order like have static objects, call-by-name, gens, etc